spec/support/test_helpers/cycle_timetable_helper.rb

```diff
@@ -1,39 +1,17 @@
 module CycleTimetableHelper
   def mid_cycle
-    previous_end_of_cycle_timetable[:apply_reopens] + 1.day
+    CycleTimetable::CYCLE_DATES.dig(2020, :apply_opens) + 1.day
```

Comment:
So the point of this was to randomise the times that tests run at to catch edge cases.

 I don't think it's a huge deal but would it have worked as expected if the dates were set to the current and next cycle, rather than previous and current i think it would have travelled forward in time rather than back?

Reply:
I believe as a result of the calendar changes, we need to work with a cycles that has a complete set of dates. We do not have all the dates for 2022 so this means we need to work with 2020 & 2021 data (or earlier)

Reply:
I can understand your point about randomising times to catch edge cases. I think it this strategy has been successful at catching edge cases it may be work re-instating. But on the other hand, having a constant date makes debugging a bit less painful (but that's just me!)

Reply:
Yea i totally get that. I wonder if we can just add the next few years dates with approximations then update when required? Ideally going forward we can standardise each date i.e. find opening on the first tues of Oct but that seems a way off.

Reply:
Hey, I'm easy about adding approximations for dates in the future. But worth considering up until this point we have approximated dates for previous years only to make some of the tests pass. I think this may be a bit of a code smell really. I think Duncan had an idea regarding creating a calendar especially for testing purposes which would resolve this however.